src/cincinnati/client.rs

```diff
@@ -0,0 +1,124 @@
+//! Asynchronous Cincinnati client.
+
+// TODO(lucab): eventually move to its own "cincinnati client library" crate
+
+#![allow(unused)]
+
+use failure::{Error, Fallible, ResultExt};
+use futures::future;
+use futures::prelude::*;
+use reqwest::r#async as asynchro;
+use reqwest::Method;
+use serde::Deserialize;
+use std::collections::HashMap;
+
+/// Cincinnati graph API path endpoint (v1).
```

Comment:
Hmm, is there a link we can add around here to some upstream Cincinnati docs about the API? (Even if just some permalink to code).